### DISCUSSION TITLE
OBS Dosn't show the latest update

### DISCUSSION THREAD

**Discussion by ApplicationCoding** [obsproject/obs-studio#5992] [General]
I opened OBS about last week and then, it showed a pop-up which is an update of the latest version "27.2". I clicked _Remind me Later_. Then, starting Sunday this week, I was about to update OBS but, when I clicked _Check For Updates_, the update window did not show. I keep clicking and clicking the button but it's still not appearing. Until today I tried to click _Check For Updates_ but it still doesn't show. I hope this problem could be solved so that others having the same problem like this can fix the problem. Thanks!

**Comment by dodgepong** [comment]
The update is on hold while we work on deploying a hotfix for an issue that was causing system crashes for certain users. This update should be coming out shortly.

https://twitter.com/OBSProject/status/1493788073875980289

**Reply by ApplicationCoding** [reply]
Oh, at least now I know about it. So, right now I opened OBS again and clicked _Check For Updates_ and... v27.2.1 is the one that showed up which is just fine since it's the latest release of OBS as of today. Thanks, @dodgepong!
